src/pages/Reimbursement.js

```diff
@@ -1,150 +1,244 @@
-						</FormText>
-						<FormFeedback invalid="true">Please upload a file in one of the following formats: .png, .jpg, .jpeg, .pdf.</FormFeedback>
-					</FormGroup>
-					<FormGroup tag="fieldset">
-						<legend id="deliveryFormatLabel">Delivery Format:</legend>
-						<FormGroup check>
-							<Label check>
-								<Input type="radio" name="radio1" id="mail" defaultChecked/>{' '}
-								Mail
-							</Label>
-						</FormGroup>
-						<FormGroup check>
-							<Label check>
-								<Input type="radio" name="radio1" id="sofs" />{' '}
-								Pick up at SOFS Office
-							</Label>
-						</FormGroup>
-					</FormGroup>
-					<Label for="submitButton">NOTE: This will be sent to your advisor for approval</Label>
-					<br />
-					<Button color="primary" id="submitButton" name="submitButton" onClick={this.handleSubmit}>Submit</Button>
-				</Form>
-			</Container>
-			);
-	}
-}
\ No newline at end of file
+        <p>
+          <b>Organization:</b> Nebraskans for Cheese
+        </p>
+        <p>
+          <b>SOFS Number:</b> 1234
+        </p>
+        <Form>
+          <FormGroup>
+            <Label for="eventSelect">Event:</Label>
+            <Input
+              type="select"
+              name="eventSelect"
+              id="eventSelect"
+              value={this.state.event}
+              onChange={this.handleChange}
+              invalid={!this.state.eventValid}
+              placeholder="Choose an event"
+            >
+              <option></option>
+              <option>Trip to the UNL Dairy Store</option>
+              <option>Trip to the Dairy Farm</option>
+              <option>Cheese Tasting</option>
+            </Input>
+            <FormFeedback invalid="true">Please select an event.</FormFeedback>
+          </FormGroup>
+          <FormGroup>
+            <Label for="eventDescription">Description:</Label>
+            <Input
+              type="textarea"
+              name="eventDescription"
+              id="eventDescription"
+              value={this.state.description}
+              onChange={this.handleChange}
+              invalid={!this.state.descriptionValid}
+            />
+            <FormFeedback invalid="true">
+              Please provide a description of what the funds were used for.
+            </FormFeedback>
+          </FormGroup>
+          <FormGroup>
+            <Label for="payTo">Pay To:</Label>
+            <Input
+              type="text"
+              name="payTo"
+              id="payTo"
+              value={this.state.payTo}
+              onChange={this.handleChange}
+              invalid={!this.state.payToValid}
+            />
+            <FormFeedback invalid="true">
+              Please provide the recipient of the reimbursement.
+            </FormFeedback>
+          </FormGroup>
+          <FormGroup>
+            <Label for="amount">Amount:</Label>
+            <Input
+              type="text"
```

Comment:
Why not use type="number" here rather than text?

Reply:
In the validation, I allow an optional $ before the number. We can remove this if we want

Reply:
I would prefer to use type number? And then if we want to show a $ we can use an addon